test/browsersync-compat.js

```diff
@@ -5,33 +5,45 @@ var pathModule = require('path');
 var root = pathModule.resolve(__dirname, '..', 'testdata') + '/';
 var processImage = require('../lib/processImage');
 
+var serverPort = '9999';
+
 expect.use(require('unexpected-http'))
     .use(require('unexpected-image'))
     .use(require('unexpected-resemble'))
     .use(require('unexpected-sinon'))
     .use(require('magicpen-prism'))
-    .addAssertion('<string> to respond with <object|number>', function (expect, subject, value) {
-        var modifiedSubject = subject.replace(' ', ' http://localhost:9999');
+    .addAssertion('<string|object> to respond with <object|number>', function (expect, subject, value) {
```

Comment:
I agree that it's a bit weird.

I think I would actually prefer to use the vanilla `to yield response satisfying` assertion (unexpected-http) and just put `http://localhost:${serverPort}` into the url in each test. That would make it easier to realize that those tests connect to a "real" server instance.